TombEngine/Objects/game_object_ids.h

```diff
@@ -998,7 +999,8 @@ enum GAME_OBJECT_ID : short
 	ID_AIR_BAR_TEXTURE,
 	ID_DASH_BAR_TEXTURE,
 	ID_SFX_BAR_TEXTURE,
-	// NOTE: 1378 - 1379 reserved for blood effects. -- Sezz 2023.05.29
+	ID_WATERFALL,
```

Comment:
We have pending blood effect pull request from Sezz, probably it's better to use another ID.